app/assets/javascripts/spree/frontend/solidus_bolt.js

```diff
@@ -1,2 +1,34 @@
 // Placeholder manifest file.
-// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
\ No newline at end of file
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js"
+
+const displayBoltInput = (paymentField, boltContainer) => {
```

Comment:
This code should be used also under admin, right? Maybe you can create a specific file for it and include it from both backend and frontend applications.

Reply:
for some reason I seem to remember you mentioning that Bolt payments will not be accessible from the Admin panel
if that memory is wrong then I will go ahead and change the file

Reply:
Yes, it is wrong . Please change the file